ArduPlane/GCS_Plane.cpp

```diff
@@ -8,6 +9,11 @@ uint8_t GCS_Plane::sysid_this_mav() const
 
 void GCS_Plane::update_vehicle_sensor_status_flags(void)
 {
+    // Compass-less plane force reporting AHRS healthy while on ground after arming
+    const auto &compass = AP::compass();
+    if (!plane.is_flying() && !compass.use_for_yaw(0)) {
```

Comment:
```suggestion
    if (!plane.is_flying() && !compass.use_for_yaw()) {
```

Reply:
there are places in the code that say dont use that....because it calls healthy something....

Reply:
you convinced me...will change